website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -10,6 +10,17 @@ To-do:
 - think about whether some of these should be outside of models
 --->
 
+| Configuration    | Description    | 
+|------------------|----------------|
+|[Incremental materialization strategies](/reference/resource-configs/redshift-configs#incremental-materialization-strategies)|The Redshift adapter supports the following incremental materialization strategies: `append` (default when `unique_key` is not defined, `merge`, `delete+insert` (default when unique_key is defined) and [`microbatch`].|
+|[Sortkey and distkey](/reference/resource-configs/redshift-configs#using-sortkey-and-distkey)|Add these keys to `{{ config(...) }}` block in model .sql file to improve query performance.|
+|[Late binding views](/reference/resource-configs/redshift-configs#late-binding-views)|"Unbinds" a view from the data it selects from.|
+|[Materialized views](/reference/resource-configs/redshift-configs#materialized-views)|See table for more information.|
```

Comment:
can we give a bit more info other than see table? e.g. supported materialized views for Redshift using configuration paramaters...